### PR TITLE
[DynaConf] Distro, Bugzilla, Shared Function and Fake Capsules settings

### PR DESCRIPTION
**Pull request by jyejare**
The Configuration files for :

1. Distro
2. Bugzilla
3. Shared Functions
4. Fake Capsules

### CODE REVIEW

File: conf/shared_function.yaml.template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  # by default 24 hours
+  SHARE_TIMEOUT: 86400
+  # If redis is used as storage, by default redis_host=localhost
+  REDIS_HOST: localhost
+  # The port redis is accessible at that redis_host, by default 6379
+  REDIS_PORT: 6379
+  # The redis db index, by default 0
+  REDIS_DB: 0
+  # The redis password index, by default None
+  # REDIS_PASSWORD:
```
Comment (omkarkhatavkar) [inline conf/shared_function.yaml.template:25]: missing validator for this

Comment (jyejare) [reply conf/shared_function.yaml.template:25]: @omkarkhatavkar Done! I also missed for `bugzilla.api_key` and I added that !

**Comment by jyejare** [comment]
@omkarkhatavkar Ready for ACK :)

File: conf/distro.yaml.template [hunk 2]
```diff
@@ -0,0 +1,6 @@
+DISTRO:
```
Comment (JacobCallahan) [inline conf/distro.yaml.template:1]: can we get at least a brief description of what these values should be?

**Review by JacobCallahan — APPROVED** [review]

**Review by mshriver — APPROVED** [review]

**Pull request merged**